instrumentation/netty/netty-3.8/src/main/java/io/opentelemetry/auto/instrumentation/netty/v3_8/client/HttpClientResponseTracingHandler.java

```diff
@@ -54,15 +54,13 @@ public void messageReceived(final ChannelHandlerContext ctx, final MessageEvent
     boolean finishSpan = msg.getMessage() instanceof HttpResponse;
 
     if (span != null && finishSpan) {
-      try (Scope scope = TRACER.withSpan(span)) {
-        DECORATE.onResponse(span, (HttpResponse) msg.getMessage());
-        DECORATE.beforeFinish(span);
-        span.end();
+      try (Scope scope = TracingContextUtils.currentContextWith(span)) {
+        TRACER.end(span, (HttpResponse) msg.getMessage());
       }
```

Comment:
we can get rid of this extra scope too
```suggestion
      TRACER.end(span, (HttpResponse) msg.getMessage());
```